server/src/controllers/booking.js

```diff
@@ -80,9 +88,59 @@ const bookingRoom = (req, res, next) => {
         );
       return bookRoom(time, roomId, userId, description);
     })
+    // eslint-disable-next-line consistent-return
     .then(({ rows }) => {
+      if (remindMe) {
+        myData = rows;
+        return myData;
+      }
```

Comment:
```suggestion
      myData = rows;
```
send the response as fast as the operation has been confirmed to work (booking the room), and sending the email can wait till it finished, no need to bind the user for few extra seconds